Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.h

```diff
@@ -43,6 +43,9 @@ namespace AZ
         public:
             AZ_COMPONENT(SceneProcessingConfigSystemComponent, "{80FE1130-91B4-44D4-869F-859BB996161A}", AZ::SceneAPI::SceneCore::SceneSystemComponent);
 
+            static constexpr const char AssetProcessorDefaultNodeSoftNameSettingsKey[] = "/O3DE/AssetProcessor/SceneBuilder/NodeSoftNameSettings";
```

Comment:
We should update the documentation to mention this setreg.

https://github.com/o3de/o3de.org/blob/development/content/docs/user-guide/assets/scene-settings/meshes-tab.md
A little over halfway down that page is the description of the LOD soft naming conventions.

Reply:
Good point. I will update the document once this PR is merged.